index.rst

```diff
@@ -39,4 +39,204 @@
 
 :tocdepth: 1
 
-Write the docs; right here.
+Data
+====
+
+In the winter of 2015, Dave Monet provided the LSST Simulations team with a
+catalog of bright stars for use in simulating the distribution of guide stars.
+The schema of this raw catalog is available in the file
+``software/README.sstrc5`` of
+`this github repository <https://github.com/lsst-sims/smtn-006>`_.  The catalog
+consists of 645 million stars with RA, Dec, proper motion and magnitudes taken
+from ca combination of 10 provenance catalogs as detailed in the README:
+USNO-B4, 2MASS, 2MASS Extended Source Catalog, WISE, PPMXL, SDSS DR7,
+PS1 Ubercal, Hipparcos + Tycho 2, APASS DR7+8, or GCVS and Lepine-Shara.
+Magnitudes provided are a subset of (B, V, u, g, r, i, z, y, J, H, K, w1,
+w2, w3, w4, SST) as detailed in the Readme.  Not all stars have the same
+subset of magnitudes measured.
+
```

Comment:
I'd suggest you provide the path to the catalog itself, or add a comment that people should only access it with these scripts.